README.md

```diff
@@ -106,7 +106,39 @@ docker run -e STORAGE_TYPE=AZURE -e BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_S
 
 Enjoy starting your journey of exploration!
 
-**Note: Uber now only provides the Espresso test feature for Android, please refer to this section for more features: [For Hydra Lab User](#for-user)**
+**Step 4. try the test procedure with simple setup**
+
+Note: Uber image now only provides the **Espresso/Instrumentation test** feature for Android, please refer to the links within "User manual" section of this page for more features: [Hydra Lab Wikis](https://github.com/microsoft/HydraLab/wiki).
```

Comment:
![image](https://user-images.githubusercontent.com/8344245/233918348-344fd347-cf41-4bd3-b6ac-c874a9eba52e.png)

Note: For Android, Uber image only supports Espresso/Instrumentation test. See “User manual” section on this page for more features.

Reply:
Applied, thx